fgrpc/dapr.go

```diff
@@ -153,6 +170,37 @@ func (d *DaprGRPCRunnerResults) prepareRequest4PubSub(o *GRPCRunnerOptions) erro
 		} else {
 			d.publishEventRequest.Data = []byte{}
 		}
+	case "publish-multi":
+		d.publishEventRequests = make([]*dapr.PublishEventRequest, numEventsInt)
+		for i := 0; i < numEventsInt; i++ {
+			d.publishEventRequests[i] = &dapr.PublishEventRequest{
+				PubsubName:      store,
+				Topic:           topic,
+				DataContentType: contentType,
+			}
+			if len(o.Payload) > 0 {
+				d.publishEventRequests[i].Data = []byte(o.Payload)
+			} else {
+				d.publishEventRequests[i].Data = []byte{}
+			}
+		}
+	case "bulkpublish":
+		d.bulkPublishRequest = &dapr.BulkPublishRequest{
+			PubsubName: store,
+			Topic:      topic,
+		}
+		d.bulkPublishRequest.Entries = make([]*dapr.BulkPublishRequestEntry, numEventsInt)
```

Comment:
Should we also enable sending `metadata` to be able to test `rawPayload` sending through App?

Reply:
We can do that, are you proposing another comparison graph for p90 and p99 with rawPayload/CloudEvent modes?

Reply:
Yes, having these two graphs may help.

Reply:
Added!